invisu.csl

```diff
@@ -448,13 +448,19 @@
         </group>
       </else-if>
       <else-if type="graphic personal_communication interview webpage" match="any"/>
-      <else-if type="article-journal article-magazine" match="any">
-        <group delimiter=" " font-style="normal" prefix=", ">
-          <group delimiter="&#160;" suffix=".">
-            <label variable="page" form="short"/>
-            <text variable="page"/>
-          </group>
-        </group>
+       <choose>
```

Comment:
not clear what you're trying to do here, but you need to define an ```<else-if>``` condition before you can put in that ```<choose>``` presumably one that corresponds to line 464 (the orphaned ```</else-if>```) below.